### DISCUSSION TITLE
TopCalc when using app_table option

### DISCUSSION THREAD

**Discussion by NicoN13** [anvilistas/tabulator#103] [Ideas]
It seems as though using topCalc and bottomCalc don't work when we use the app_table option to display a client_readable view of an anvil data table. 

It would be great if we could use these column options as well. I understand the point of using app_table is to lazy load data from a table, so perhaps using topCalc (which then needs to fetch each row) kind of defeats the purpose. 

Perhaps @s-cork can think of an ingenious way of still fetching the data in a lazy manner, but still having the ability to have a topCalc which displays the column sum perhaps? Not sure if this is possible but it would certainly be extemely useful.

**Comment by s-cork** [comment]
I can't think of anything off the top of my head. 
Anyway you'd want to code that sum would require a full fetch of all rows.

**Reply by NicoN13** [reply]
Yeah, so basically it would fetch the total sum of a specified column, without fetching the rest of the data and only load those as we scroll them into view. For now I'll try do a work around.
